src/utils.js

```diff
@@ -1,3 +1,5 @@
+// @flow
+
```

Comment:
Will `@flow` be added to more files in a future PR?

Reply:
Yep! That's the plan a bunch of the files need work for flow to work well.